vote/views.py

```diff
@@ -13,11 +13,17 @@
 def poll(request, poll_identifier):
     poll = get_object_or_404(Poll, identifier=poll_identifier)
     token = request.GET.get('token', "Please enter your voting token")
+    try:
+        token_object = Token.objects.get(token_string=token)
+        error_message = None
+    except Token.DoesNotExist:
+        error_message = "Token from link was already used"
```

Comment:
'This token is invalid. Maybe you already voted?' would be better IMO. 
Also: Would it be also valid to access the site without a token? If yes, this case would then trigger an error as of now.